AUTHORS

```diff
@@ -2,5 +2,5 @@ The PyFCP applications and library were written
 by David McNab (aum) in May-June, 2006.
 
 Contact the author at:
-  david@freenet.org.nz
+  david@rebirthing.co.nz
```

Comment:
From the man pages. Then I checked http://rebirthing.co.nz in search of this email adress. Did not find it but find a picture of David McNab on the first page. I write him to be certain. No reply yet but I did not receive any error. So this email is most probably working.